src/test/data/ManualTesting/testScript.md

```diff
@@ -0,0 +1,233 @@
+
+###2. Mark multiple tasks
+Type `mark 1 3-5`.
+####Result
+Tasks with indices 1, 3, 4 and 5 are marked as completed, shifting from **Do It Soon** column to **Done** column, and their completion times are shown below their task names. Also, they are all highlighted in purple borders.
+
+## Unmark
+To unmark a task as completed, you have to start your command with the keyword `unmark`.
+
+>Here is the *format*:
+
+>* `unmark <id>...` - unmark all the tasks identified by `<id>`(s) as uncompleted. Each `<id>` must be a positive number and in the most recent to-do list displayed.
+
+###1. Unmark a single task
+Type `mark 36`.
+####Result
+Task with index 36 is marked as completed, shifting from **Done** column to **Do It Soon** column, and there is no more completion time under the task name. Also, it is highlighted in purple borders.
+
+###2. Unmark multiple tasks
+Type `mark 37-40`.
+####Result
+Tasks with indices 37, 38, 39 and 40 are unmarked as uncompleted, shifting from **Done** column to **Do It Soon** column, and there are no more completion times under their task names. Also, they are all highlighted in purple borders.
+
+## Undo
+1. Type `undo`.
+2. Press <kbd>Ctrl Z</kbd>
+
+####Result
+1. The previous unmarked tasks are marked again, and the affected tasks are highlighted in purple borders.
+2. The previous marked task is marked again, and the affected task is highlighted in purple borders.
+
+## Find
+>Here is the *format*:
+
+>* `find <keyword>...` - filter out all tasks containing any of the keyword(s) given
+
+Type `find cs2103`
+####result
+Tasks containing `cs2103` in their names will be listed out.
+
+## List
+After you are done with searching for tasks, you can use `list` to exit your find results and see a list of tasks. Alternatively, you can press <kbd>ESC</kbd> to go exit.
+
+1. Type `list`
+2. Type `find cs2103`
+3. Press <kbd>ESC</kbd>
+
+####Result
+1. After the first step, agendum will exit the find results and go back to list all the tasks.
+2. After the second step, agendum will list out tasks containing `cs2103` in their names.
+3. After the third step, agendum will exit the find results and go back to list all the tasks.
+
+## Alias
+To alias a command, you have to start your command with the keyword `alias`.
+>Here is the *format*:
+
+>* `alias <original command> <your command>` - `<your command>` must be a single alphanumeric word, and it cannot be a original-command or already aliased to another command. `<original command>` must be a command word that is specified in the Command Summary section
+
+1. Type `alias schedule s`
+2. Type `s 1 by 10pm`
+
+####Result
+1. After the first step, `s` is created as an alias command for `schedule`, and you can now use both `s` and `schedule` to reschedule a task.
+2. After the second step, task with index 1 is rescheduled to be by today 10pm.
+
```

Comment:
just wondering. are we supposed to show that we handle edge cases
e.g. alias mark add